pkg/ruler/rules/rules.proto

```diff
@@ -13,7 +13,11 @@ import "github.com/cortexproject/cortex/pkg/ingester/client/cortex.proto";
 option (gogoproto.marshaler_all) = true;
 option (gogoproto.unmarshaler_all) = true;
 
-// RuleGroupDesc is a proto representation of a cortex rule group
+// RuleGroupDesc is a proto representation of a cortex rule group.
+// The options field in the RuleGroupDesc can be used to extend Cortex Ruler
+// functionality without having to repeatedly redefine the proto description.
+// It can also be leveraged to create custom `ManagerOpts` based on rule configs
+// which can then be passed to the Prometheus Manager.
```

Comment:
Please be the description of a field to a comment in the line above the field definition itself.

Reply:
Addressed :)